docs/reference/security/authentication/saml-guide.asciidoc

```diff
@@ -328,24 +328,15 @@ groups:: _(Recommended)_
 +
 [NOTE]
 ====
-Some IdPs are configured to send the `groups` list as a comma-separated string,
-but {es} can't parse this string into an array of groups. To map this SAML
-attribute to the `attributes.groups` setting in the {es} realm, a cluster
```

Comment:
Total language nit: our docs generally use active voice vs. passive, so I'd go with "you can configure a string delimiter using the `attribute_delimiters.group`" setting.